composer.json

```diff
@@ -10,20 +36,20 @@
   "require": {
     "dompdf/dompdf" : "0.7.*",
     "electrolinux/phpquery": "^0.9.6",
-    "symfony/config": "~2.5.0",
-    "symfony/dependency-injection": "~2.5.0",
-    "symfony/event-dispatcher": "~2.5.0",
-    "symfony/filesystem": "~2.5.0",
-    "symfony/process": "~2.5.0",
-    "psr/log": "1.0.0",
-    "symfony/finder": "~2.5.0",
+    "symfony/config": "^2.5.0",
```

Comment:
ah, yeah. https://semver.mwl.be/#?package=symfony%2Fprocess&version=~2.5%20%7C%7C%20~3.0&minimum-stability=stable that looks like it'd be the trick (for also supporting 8.4.x)